.github/CODEOWNERS

```diff
@@ -14,6 +14,7 @@
 /.github                            @benesch
 /bin
 /ci                                 @MaterializeInc/testing
+/ci/test/lint-deps.toml             @danhhz @benesch
```

Comment:
@benesch you okay with this? I figured we should have at least two on it in case I'm on vacation

Reply:
Yep!